components/common/ShareButtons.js

```diff
@@ -5,12 +5,16 @@ import { faFacebook, faTwitter } from '@fortawesome/free-brands-svg-icons';
 
 const ShareButtons = ({ className }) => (
     <div className={ className }>
-        <button type="button" className="btn btn-sm btn-primary mr-2">
-            <FontAwesomeIcon icon={ faFacebook } className="mr-1" /> Share
-        </button>
-        <button type="button" className="btn btn-sm btn-primary" style={{backgroundColor: '#1b95e0', borderColor: '#1b95e0'}}>
-            <FontAwesomeIcon icon={ faTwitter } className="mr-1" /> Tweet
-        </button>
+        <a href="https://www.facebook.com/sharer/sharer.php?u=https://freefrom-map-frontend.herokuapp.com/" target="_blank" rel="noopener noreferrer">
+            <button type="button" className="btn btn-sm btn-primary mr-2">
+                <FontAwesomeIcon icon={ faFacebook } className="mr-1" /> Share
+            </button>
+        </a>
```

Comment:
Since the button doesn't do anything, we should get rid of it. I'm p sure this is going to confuse screenreaders:
```suggestion
        <a
            className="btn btn-sm btn-primary mr-2"
            href="https://www.facebook.com/sharer/sharer.php?u=https://freefrom-map-frontend.herokuapp.com/"
            target="_blank"
            rel="noopener noreferrer"
        >
            <FontAwesomeIcon icon={ faFacebook } className="mr-1" /> Share
        </a>
```
Also, maybe this is over-engineering too early, but we can use [Location](https://developer.mozilla.org/en-US/docs/Web/API/Location) to automatically set the href depending on what page the user is on:
```suggestion
        <a
            className="btn btn-sm btn-primary mr-2"
            href={`https://www.facebook.com/sharer/sharer.php?u=${window.location.href}`}
            target="_blank"
            rel="noopener noreferrer"
        >
            <FontAwesomeIcon icon={ faFacebook } className="mr-1" /> Share
        </a>
```
`window.location.href` includes the path (e.g. `https://github.com/RagtagOpen/freefrom-map-frontend/pull/137/files`), but if you don't want that, you can use `window.location.origin`, which leaves out the path (e.g. `https://github.com/`)
```suggestion
        <a
            className="btn btn-sm btn-primary mr-2"
            href={`https://www.facebook.com/sharer/sharer.php?u=${window.location.origin}`}
            target="_blank"
            rel="noopener noreferrer"
        >
            <FontAwesomeIcon icon={ faFacebook } className="mr-1" /> Share
        </a>
```